res_users_password_security/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 
     'name': 'Password Security',
     "summary": "Allow admin to set password security requirements.",
-    'version': '9.0.1.0.1',
+    'version': '9.0.1.0.2',
```

Comment:
Once merged, people that are using the `9.0.1.0.1` version from my repo will need a way to indicate the change won't they? I honestly have no idea how the Odoo App store works, but I figured these version numbers has some relevance to the auto update?